src/app/sign-in/components/social/social.component.html

```diff
@@ -4,10 +4,10 @@
   class="institution mat-button-wrap-text-with-icon mat-elevation-z0"
   type="button"
 >
-  <mat-icon svgIcon="institutional-access"></mat-icon>
-  <ng-container
-    i18n="@@ngOrcid.signin.institutionalAccount"
-  >
+  <a>
```

Comment:
Is a little odd to use an anchor tag, to just contain an element (not linked to anything). 
But I don't think this is critical we can keep it

Reply:
I will remove it

Reply:
I will erase it